src/main/scala/no/ndla/conceptapi/service/search/IndexService.scala

```diff
@@ -241,7 +245,7 @@ trait IndexService {
               textField(s"$fieldName.${langAnalyzer.lang}")
                 .fielddata(false)
                 .analyzer(langAnalyzer.analyzer)
-                .fields(keywordField("raw")))
+                .fields(keywordField("raw"), keywordField("lower").normalizer("lower")))
```

Comment:
Tvinger dette til å være lowercase for å kunne matche på søk uavhengig av casing.